CMakeLists.txt

```diff
@@ -259,8 +259,8 @@ ELSE()
   MESSAGE(STATUS "Disabling unity build.")
 ENDIF()
 
-add_executable(gwb-dat "${CMAKE_CURRENT_SOURCE_DIR}/app/main.cc")
-add_executable(gwb-grid "${CMAKE_CURRENT_SOURCE_DIR}/visualization/main.cc")
+add_executable(gwb-dat "${CMAKE_CURRENT_SOURCE_DIR}/source/gwb-dat/main.cc")
+add_executable(gwb-grid "${CMAKE_CURRENT_SOURCE_DIR}/source/gwb-grid/main.cc")
```

Comment:
I think it makes sense to have source files in directories that match the executable's name.